src/tokens/token-groups/legacy-tokens.json

```diff
@@ -1,7 +1,9 @@
 {
   "override-loading-z-index": "514",
   "choice-size": "20px",
-  "icon-size": "10px",
+  "icon-size-small": "10px",
+  "icon-size-medium": "20px",
+  "icon-size-large": "30px",
```

Comment:
Is there any usage of `large`? If not, maybe we don't need to include it?

Reply:
Yeah, lets not add a value that is not used.